community/subteams.md

```diff
@@ -0,0 +1,315 @@
+  conda-forge.
+- Maintain the infrastructure for `CFEP-13`.
+- Maintain automated ways to recover and reprovision conda-forge
+  systems.
+- Perform ongoing security maintenance tasks.
+- Provision new members of <span class="title-ref">core</span> with
+  access to all conda-forge systems, credentials and keys.
+
+This team will operate via the following rules:
+
+- Only members of `core` can be on this sub-team.
+- This sub-team will report its activity to core at all `core` meetings.
+- This sub-team will consult with core at `core` meetings regarding all
+  planned activity.
+- This sub-team will not limit the access by `core` members to any
+  conda-forge resources including but not limited to its systems,
+  credentials, keys, or service accounts.
+
+### Members
+
+- Matthew R. Becker \<<becker.mr@gmail.com>\>
+- Christopher J. "CJ" Wright \<<cjwright4242@gmail.com>\>
+- Anthony Scopatz \<<scopatz@gmail.com>\>
+
+## Finance Sub-Team
+
+### Role
+
+The purpose of the finance sub-team is to provide a point of contact for
+financial and budgetary issues. This includes keeping core aware of the
+current conda-forge balance via `core` meetings and facilitating
+dispersal of funds.
+
+### Charter
+
+Static
+
+### Responsibility
+
+The core responsibilities of this team are as follows:
+
+- Keep core aware of budgetary and financial matters pertaining to
+  conda-forge.
+- Facilitate dispersal of funds.
+- Give core updates via the standing budget item at core meetings.
+- Work closely with the NumFOCUS point of contact to ensure smooth
+  financial operations.
+
+This team will operate via the following rules:
+
+- Only members of `core` can be on this sub-team.
+- This sub-team will report its activity to core at all attended `core`
+  meetings.
+- This sub-team will not approve or deny access to funds unless
+  instructed to via the method specified by the conda-forge governance
+  document.
+
+### Members
+
+- Christopher J. "CJ" Wright \<<cjwright4242@gmail.com>\>
+- Filipe Fernandes \<<ocefpaf@gmail.com>\>
+- Eric Dill \<<ericdill@pm.me>\>
+
+## Bot Sub-Team
+
+### Role
+
+Develop and manage the conda-forge migration bot and all related
+tooling. Also manages and deploys migrations themselves.
+
+### Charter
+
+Dynamic
+
+### Responsibility
+
+The migration and autotick bot is now a central part of the conda-forge
+ecosystem. This subteam has the right and responsibility to manage and
+develop the general operation of the bot. This includes building new
+migrators, fixing migration related bugs, and tooling. Example
+migrations that can happen include:
+
+- Compiler bumps
+- Python version bump
+- R version bump
+- Build number bumps of the ecosystem when a pinned package version
+  updates and there is a binary incompatibility which necessitates
+  downstream rebuilds.
+- Automatically version bumping of feedstocks when the package releases
+  a new version.
+
+For large scale (affecting \>20% of packages) this sub-team will inform
```

Comment:
Do we still need to escape the `<>` symbols with the renderer set to autodetect by file extension?

Reply:
I think this was like this from the beginning? 🤔 but sure